frontend/app/routes/_gcweb-app.personal-information._index.tsx

```diff
@@ -55,18 +57,36 @@ export default function PersonalInformationIndex() {
         </PersonalInformationSection>
         <PersonalInformationSection
           footer={
-            <Link id="change-home-address-button" className="btn btn-primary btn-lg" to="/personal-information/address/edit">
-              {t('personal-information:index.change-addresses')}
+            <Link id="change-home-address-button" className="btn btn-primary btn-lg" to="/personal-information/home-address/edit">
+              {t('personal-information:index.change-home-address')}
+            </Link>
+          }
+          title={t('personal-information:index.home-address')}
+          icon="glyphicon-map-marker"
+        >
+          <dl>
+            <dd className="whitespace-pre-line">{homeAddressInfo?.address}</dd>
+            <dd className="whitespace-pre-line">{homeAddressInfo?.city}</dd>
+            <dd className="whitespace-pre-line">{homeAddressInfo?.postalCode}</dd>
+            <dd className="whitespace-pre-line">{homeAddressInfo?.province}</dd>
+            <dd className="whitespace-pre-line">{homeAddressInfo?.country}</dd>
+          </dl>
```

Comment:
We should use the `<Address>` component to render the home and mailing addresses rather than these description lists.

Reply:
Done